enterprise/templates/enterprise/enterprise_course_enrollment_page.html

```diff
@@ -55,26 +55,29 @@ <h3 class="course-confirmation-title">${ confirmation_text }</h3>
           </div>
           <form>
             <div class="caption">${ select_mode_text }</div>
+            % for course_mode in course_modes:
             <div class="radio-block">
               <div class="radio">
-                <input type="radio" name="data_sharing_consent" id="radio1" checked="checked"/>
+                <input type="radio" name="data_sharing_consent" id="radio${loop.index}"
+                       % if loop.first:
+                       checked="checked"
+                       % endif
+                       />
```

Comment:
Good point - possibly change this to a hidden field with a default value if there's just one item.